backend/db/config.py

```diff
@@ -79,22 +79,17 @@ def get_pg_connect_url(local=False):
 REFRESH_JOB_MAX_HRS = 6  # see is_refresh_active() for documentation on this var
 CORE_CSET_TABLES = ['code_sets', 'concept_set_container', 'concept_set_version_item', 'concept_set_members']
 CORE_CSET_DEPENDENT_TABLES = [
+    # tables
     'cset_members_items',
     'codeset_ids_by_concept_id',
     'concept_ids_by_codeset_id',
     'members_items_summary',
     'codeset_counts',
     'all_csets',
+    # views
     # 'csets_to_ignore',
     'cset_members_items_plus',
 ]
-# todo: try to auto detect what is a view
-VIEWS = [
```

Comment:
## Delete: Views related config
This bit is either now obsolete or replaced by `list_views()`.